library/pdns_zone.py

```diff
@@ -244,30 +260,44 @@ def zone_add_slave(module, base_url, zone, masters, comment):
         return False
 
     if kind == 'MASTER' or kind == 'NATIVE':
-        module.fail_json( msg="zone %s is %s. Cannot convert to slave" % (zone, kind))
+        module.fail_json(
+            msg="zone %s is %s. Cannot convert to slave" % (zone, kind)
+        )
 
     masters = masters.split(',')
 
     data = {
-        'kind'      : 'Slave',
-        'masters'   : masters,
-        'name'      : zone,
-        'comments'      : [{
-                            'name'  : zone,
-                            'type'  : 'SOA',
-                            'account' : '',
-                            'content' : comment,
-                          }],
+        'kind': 'Slave',
+        'masters': masters,
+        'name': zone,
+        'comments': [
+            {'name': zone, 'type': 'SOA', 'account': '', 'content': comment}
+        ],
     }
     payload = json.dumps(data)
 
-    response, info = fetch_url(module, base_url, data=payload, headers=headers, method='POST')
+    response, info = fetch_url(
+        module, base_url, data=payload, headers=headers, method='POST'
+    )
     if info['status'] != 201:
-        module.fail_json(msg="failed to create slave zone %s at %s: %s" % (zone, base_url, info['msg']))
+        module.fail_json(
+            msg="failed to create slave zone %s at %s: %s"
+            % (zone, base_url, info['msg'])
+        )
 
     return True
 
-def zone_add_master(module, base_url, zone, soa_rdata, ns_rrset, comment, ttl=60, wantkind='Master'):
```

Comment:
I have to ask again: what's wrong here? I don't much care for the parameters on a new line each

Reply:
line length